packages/world/src/World.sol

```diff
@@ -71,10 +72,12 @@ contract World is StoreData, IWorldKernel {
     }
 
     // The World can only be initialized once
-    if (InstalledModules._get(CORE_MODULE_NAME, keccak256("")) != address(0)) {
+    if (WorldInitialized.get()) {
       revert World_AlreadyInitialized();
     }
 
+    WorldInitialized.set(true);
```

Comment:
What if we store the core module's address here? Almost the same cost (since it's one storage write) but might be interesting to know which core module was used to init the world

Reply:
Sure, that seems like a reasonable thing to keep around.